src/dev_deps.ts

```diff
@@ -3,5 +3,8 @@ export {
   assertExists,
   assertRejects,
   assertStringIncludes,
-} from "https://deno.land/std@0.132.0/testing/asserts.ts";
-export * from "https://deno.land/x/mock_fetch@0.3.0/mod.ts";
+  assertThrows,
+  fail,
+} from "https://deno.land/std@0.152.0/testing/asserts.ts";
+export * as mock from "https://deno.land/std@0.152.0/testing/mock.ts";
```

Comment:
decided it's probably safer to name `*` exports like this to avoid collisions. Might be worth doing the same w/ the assert import, but didn't wanna include all of that in this PR too.